src/components/Editor/Editor.js

```diff
@@ -91,9 +91,10 @@ class Editor extends React.Component {
 
   componentWillReceiveProps(nextProps) {
     const { title, topics, body, reward, upvote } = this.props;
+
     if (
       title !== nextProps.title ||
-      topics !== nextProps.topics ||
+      JSON.stringify(topics) !== JSON.stringify(nextProps.topics) ||
```

Comment:
As you said `JSON.stringify` is not the best option there. Can you use lodash's `isEqual` method for comparsion?

Reply:
Added